### PR TITLE
Adding Hivemetastore V2.4.7 helm chart

### PR DESCRIPTION
**Pull request by apurwaj2**
Configuration for derby db.

### CODE REVIEW

**Review by akshayharyani — CHANGES_REQUESTED** [review]

File: charts/metastore-2.4.7/hivemeta-chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+    - "--"
+
+  env:
+    - name: KUBERNETES_CONTAINER
+      value: 'true'
+    - name: LOG_LEVEL
+      value: INFO
+    - name: SSH_PORT
+      value: '7778'
+    - name: "CLUSTER_CREATE_TIME"
```
[the remark this answers is not in the document]
Comment (apurwaj2) [reply charts/metastore-2.4.7/hivemeta-chart/values.yaml]: Done

File: charts/metastore-2.4.7/hivemeta-chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+createDefaultPullSecret: false
+defaultPullSecret: imagepull
+
+automountServiceAccountToken: true
+
+#sparkVersion for the applications
+sparkVersion: spark-2.4.7
+
+#HPE tenant namespace info
+tenantNameSpace: sampletenant
```
[the remark this answers is not in the document]
Comment (apurwaj2) [reply charts/metastore-2.4.7/hivemeta-chart/values.yaml]: Done.

**Review by AlexNavara — DISMISSED** [review]
lgtm

**Review by akshayharyani — APPROVED** [review]
LGTM

**Review by AlexNavara — APPROVED** [review]

**Pull request merged**